addons/mrp/migrations/8.0.1.1/post-migrate.py

```diff
@@ -95,17 +95,36 @@ def fix_domains(cr, pool):
 
 def update_stock_moves(cr, pool):
     stock_move_obj = pool['stock.move']
-    sm_ids = stock_move_obj.search(cr, uid, [])
     mrp_production_obj = pool['mrp.production']
     location_obj = pool['stock.location']
-    location_id = location_obj.search(cr, uid, [('name', '=', 'Production')])
+
+    # TODO: improve production location search, using other criteria than name
+    location_ids = location_obj.search(cr, uid, [('name', '=', 'Production')])
+    if not location_ids:
+        # This function is useless
+        return True
+
+    # TODO: make this script operationnel in multicompany context
+    location_id = location_ids[0]
+    sm_ids = stock_move_obj.search(cr, uid, [
+        ('location_dest_id', '=', location_id)])
```

Comment:
If you look the moves for all the found locations, you would have it multi-company aware, doesn't it?